Assets/Scripts/CameraShake.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEngine;
+using System.Collections;
+
+// taken from https://gist.github.com/ftvs/5822103
+
+public class CameraShake : MonoBehaviour
+{
+	// Transform of the camera to shake. Grabs the gameObject's transform
+	// if null.
+	public Transform camTransform;
+
+	// How long the object should shake for.
+	public float shakeDuration = 0f;
+
+	// Amplitude of the shake. A larger value shakes the camera harder.
+	public float shakeAmount = 0.7f;
+	public float decreaseFactor = 1.0f;
+
+	Vector3 originalPos;
+
+	void Awake()
+	{
+		if (camTransform == null)
+		{
+			camTransform = GetComponent(typeof(Transform)) as Transform;
+		}
+	}
+
+	void OnEnable()
+	{
+		originalPos = camTransform.localPosition;
+	}
+
+	void Update()
+	{
+		if (shakeDuration > 0)
+		{
+			camTransform.localPosition = originalPos + Random.insideUnitSphere * shakeAmount;
+
+			shakeDuration -= Time.deltaTime * decreaseFactor;
+		}
+		else
+		{
+			shakeDuration = 0f;
+			camTransform.localPosition = originalPos;
+		}
+	}
```

Comment:
For our scope of game it doesn't really matter, but I would imagine there would be a more "optimal" way of doing this without doing a couple of operations in each refresh, since there's a clear event that determines the start of the shake.

Reply:
Thats a good point. everything here was copied and pasted, but we should keep this in mind if we need to optimize performance.